puppeteerInterface.js

```diff
@@ -75,15 +75,15 @@ class PuppeteerInterface {
      */
     async addRules (rules) {
         await this.ready
-        await this.backgroundWorker?.evaluate(async rules => {
+        await this.backgroundWorker?.evaluate(async addRules => {
```

Comment:
Does ESLint not consider an `addRules` parameter to clash with the function name? (If not, then fine by me.)

Reply:
No, becasue `addRules` is a property of the class, not a variable in this scope.